Tests/AppTests/Util.swift

```diff
@@ -24,8 +24,8 @@ import XCTest
 
 private var _schemaCreated = false
 
-func setup(_ environment: Environment, resetDb: Bool = true) throws -> Application {
-    let app = Application(.testing)
+func setup(_ environment: Environment, resetDb: Bool = true) async throws -> Application {
+    let app = Application(environment)
```

Comment:
This was a potential bug but we only ever call it with `.testing` as a parameter.